test-utils/src/address_tree_rollover.rs

```diff
@@ -174,15 +180,15 @@ pub async fn assert_rolled_over_address_merkle_tree_and_queue<R: RpcConnection>(
         new_queue_pubkey,
     );
 
-    let struct_old = unsafe {
-        &*(old_loaded_mt_account.merkle_tree_struct.as_ptr()
-            as *mut ConcurrentMerkleTree<Poseidon, { ADDRESS_MERKLE_TREE_HEIGHT as usize }>)
-    };
-    let struct_new = unsafe {
-        &*(new_loaded_mt_account.merkle_tree_struct.as_ptr()
-            as *mut ConcurrentMerkleTree<Poseidon, { ADDRESS_MERKLE_TREE_HEIGHT as usize }>)
-    };
-    assert_rolledover_merkle_trees(struct_old, struct_new);
+    // let struct_old = unsafe {
+    //     &*(old_loaded_mt_account.merkle_tree_struct.as_ptr()
+    //         as *mut ConcurrentMerkleTree<Poseidon, { ADDRESS_MERKLE_TREE_HEIGHT as usize }>)
+    // };
+    // let struct_new = unsafe {
+    //     &*(new_loaded_mt_account.merkle_tree_struct.as_ptr()
+    //         as *mut ConcurrentMerkleTree<Poseidon, { ADDRESS_MERKLE_TREE_HEIGHT as usize }>)
+    // };
+    // assert_rolledover_merkle_trees(struct_old, struct_new);
```

Comment:
leftover?

Reply:
I recovered this test. Just forgot to do so before